### ISSUE TITLE
Implement about:config

### ISSUE THREAD

**Issue by joeparkerz** [brave/brave-browser#1124]
Implement about:config like it is for Firefox. Currently it is too limited to change anything.

**Comment by Liunkae** [comment]
Have you tried `chrome://flags/`?

**Comment by DevilEars** [comment]
Yeah this would be great! I am trying to prevent tabs from opening right next to the current active one, and open up at the end of the browsing window instead. There is (still.. for now) config to do this, but it seems Brave has disabled or masked the about:config menu. This is not a wise move IMO.

**Comment by bsclifton** [comment]
@DevilEars where is this setting in Chrome? For sure we don't have an about:config screen... so you're limited to what's available in brave://settings (although there are other internal pages you can view; see brave://chrome-urls/ for the full list)

**Comment by DevilEars** [comment]
Wow, thanks for the speedy reply. 

My bad, the setting is in Firefox's about:config. The setting is specifically:

`browser.tabs.insertRelatedAfterCurrent=false`

**Comment by DevilEars** [comment]
> Have you tried `chrome://flags/`?

Yes. It doesn't have the same functionality as Firefox's about:config at all
